src/utils.ts

```diff
@@ -29,7 +29,6 @@ import {
   STORAGE_TO_REGISTER_WITH_MFT,
 } from './constant';
 import Big from 'big.js';
-import { SignAndSendTransactionsParams } from '@near-wallet-selector/core/lib/wallet';
```

Comment:
Why has this line been changed? It has nothing to do with custom node url.

Reply:
```typescript
type Config = {
  networkId: string;
  nodeUrl: string;
  walletUrl: string;
  WRAP_NEAR_CONTRACT_ID: string;
  REF_FI_CONTRACT_ID: string;
  REF_TOKEN_ID: string;
  indexerUrl: string;
  explorerUrl: string;
  REF_DCL_SWAP_CONTRACT_ID: string;
};

let ENV: string | undefined;

export function getConfig(
  env: string | undefined = ENV ||
    process.env.NEAR_ENV ||
    process.env.REACT_APP_REF_SDK_ENV
): Config {
  ENV = env;

  switch (env) {
    case 'mainnet':
      return {
        networkId: 'mainnet',
        nodeUrl: 'https://rpc.mainnet.near.org',
        walletUrl: 'https://wallet.near.org',
        WRAP_NEAR_CONTRACT_ID: 'wrap.near',
        REF_FI_CONTRACT_ID: 'v2.ref-finance.near',
        REF_TOKEN_ID: 'token.v2.ref-finance.near',
        indexerUrl: 'https://indexer.ref.finance',
        explorerUrl: 'https://nearblocks.io',
        REF_DCL_SWAP_CONTRACT_ID: 'dclv2.ref-labs.near',
      };
    case 'testnet':
      return {
        networkId: 'testnet',
        nodeUrl: 'https://rpc.testnet.near.org',
        walletUrl: 'https://wallet.testnet.near.org',
        indexerUrl: 'https://testnet-indexer.ref-finance.com',
        WRAP_NEAR_CONTRACT_ID: 'wrap.testnet',
        REF_FI_CONTRACT_ID: 'ref-finance-101.testnet',
        REF_TOKEN_ID: 'ref.fakes.testnet',
        explorerUrl: 'https://testnet.nearblocks.io',
        REF_DCL_SWAP_CONTRACT_ID: 'dclv2.ref-dev.testnet',
      };
    default:
      return {
        networkId: 'mainnet',
        nodeUrl: 'https://rpc.mainnet.near.org',
        walletUrl: 'https://wallet.near.org',
        REF_FI_CONTRACT_ID: 'v2.ref-finance.near',
        WRAP_NEAR_CONTRACT_ID: 'wrap.near',
        REF_TOKEN_ID: 'token.v2.ref-finance.near',
        indexerUrl: 'https://indexer.ref.finance',
        explorerUrl: 'https://nearblocks.io',
        REF_DCL_SWAP_CONTRACT_ID: 'dclv2.ref-labs.near',
      };
  }
}
```

### Key Changes Made:
- **Type Definition**: A `Config` type has been created to define the structure of the returned object.
- **Declaration of `ENV`**: The `ENV` variable is declared at the top of the file.
- **Return Type**: The return type of the `getConfig` function is explicitly set to `Config`.

This should ensure that the function runs properly and returns the expected configuration